models/comments.models.js

```diff
@@ -10,4 +9,22 @@ function fetchCommentsById(params) {
     });
 }
 
-module.exports = {fetchCommentsById};
\ No newline at end of file
+function addCommentByArticleId(comment, params) {
+    if(!comment.body.length) return Promise.reject({status: 400, msg: 'empty comment'});
+    return db.query(`SELECT * FROM users WHERE username = $1`, [comment.username])
+    .then(({rows}) => {
+        if(!rows.length) return Promise.reject({status: 404, msg: 'user doesn\'t exist'})
+        return db.query(`SELECT * FROM articles WHERE article_id = $1`, [params.article_id])
+    })
+    .then(({rows}) => {
+        if(!rows.length) return Promise.reject({status: 404, msg: 'article doesn\'t exist'});
+        return db.query(`
+        INSERT INTO comments (body, author, created_at, article_id, votes)
```

Comment:
you should not need to post into the created_at and votes rows as these have default values(or at least should do!) in your database schema - these can be removed.